pinot-common/src/main/java/com/linkedin/pinot/common/query/ServerQueryRequest.java

```diff
@@ -33,10 +35,11 @@
  * etc.
  *
  */
-public class QueryRequest {
-  private static final Logger LOGGER = LoggerFactory.getLogger(QueryRequest.class);
+public class ServerQueryRequest {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ServerQueryRequest.class);
 
   private final InstanceRequest instanceRequest;
+  private final FilterQueryTree filterQueryTree;
```

Comment:
I refrain from changing member variable naming convention to make it simpler to review the code. Better to have pure clean ups without any functional change as separate PR's.